### ISSUE TITLE
yolov3.onnx problem

### ISSUE THREAD

**Issue by tianxinyu0509** [triton-inference-server/server#2554]
Xavier has a Deepstraem-Triton environment on it. I copied the densenet_onnx model on Xavier to X86. The model was then deployed on X86 using Triton-Inference-Server without any problems.
![图片](https://user-images.githubusercontent.com/36063678/108961046-c026f680-76b1-11eb-80f3-5b99d80bf82d.png)
But when I replace my model with yolov3, it's not going to be right. The only difference is the onnx file. Can someone tell me what needs to be changed in the model file?
I don't know what error happened when I used yolov3-onnx. Here is my terminal command and the result. If you need a complete terminal display please ask me.
`nvidia-docker run -it --name trt_server --rm --shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 -p 8000:8000 -p 8001:8001 -p8002:8002 -v /home/txy/SDQZtxy/yolov3-tensorrt:/models nvcr.io/nvidia/tritonserver:20.12-py3 tritonserver --model-repository=/models --log-verbose=1 --strict-model-config=false`

`I0224 07:15:47.023157 1 server.cc:490] 
+-------------+-----------------------------------------------------------------+------+
| Backend     | Config                                                          | Path |
+-------------+-----------------------------------------------------------------+------+
| pytorch     | /opt/tritonserver/backends/pytorch/libtriton_pytorch.so         | {}   |
| tensorflow  | /opt/tritonserver/backends/tensorflow1/libtriton_tensorflow1.so | {}   |
| onnxruntime | /opt/tritonserver/backends/onnxruntime/libtriton_onnxruntime.so | {}   |
+-------------+-----------------------------------------------------------------+------+

I0224 07:15:47.023173 1 model_repository_manager.cc:428] BackendStates()
I0224 07:15:47.023192 1 server.cc:533] 
+-------------+---------+--------+
| Model       | Version | Status |
+-------------+---------+--------+
| yolov3_onnx | 1       | READY  |
+-------------+---------+--------+

I0224 07:15:47.023268 1 tritonserver.cc:1620] 
+----------------------------------+------------------------------------------------------------------------------------------------------------------------------------+
| Option                           | Value                                                                                                                              |
+----------------------------------+------------------------------------------------------------------------------------------------------------------------------------+
| server_id                        | triton                                                                                                                             |
| server_version                   | 2.6.0                                                                                                                              |
| server_extensions                | classification sequence model_repository schedule_policy model_configuration system_shared_memory cuda_shared_memory binary_tensor |
|                                  | _data statistics                                                                                                                   |
| model_repository_path[0]         | /models                                                                                                                            |
| model_control_mode               | MODE_NONE                                                                                                                          |
| strict_model_config              | 0                                                                                                                                  |
| pinned_memory_pool_byte_size     | 268435456                                                                                                                          |
| cuda_memory_pool_byte_size{0}    | 67108864                                                                                                                           |
| min_supported_compute_capability | 6.0                                                                                                                                |
| strict_readiness                 | 1                                                                                                                                  |
| exit_timeout                     | 30                                                                                                                                 |
+----------------------------------+------------------------------------------------------------------------------------------------------------------------------------+

I0224 07:15:47.023293 1 server.cc:215] Waiting for in-flight requests to complete.
I0224 07:15:47.023302 1 model_repository_manager.cc:555] AsyncLoad() 'yolov3_onnx'
I0224 07:15:47.023306 1 model_repository_manager.cc:730] TriggerNextAction() 'yolov3_onnx' version 1: 2
I0224 07:15:47.023311 1 model_repository_manager.cc:813] Unload() 'yolov3_onnx' version 1
I0224 07:15:47.023314 1 model_repository_manager.cc:820] unloading: yolov3_onnx:1
I0224 07:15:47.023341 1 model_repository_manager.cc:392] LiveBackendStates()
I0224 07:15:47.023348 1 server.cc:230] Timeout 30: Found 1 live models and 0 in-flight non-inference requests
I0224 07:15:47.023352 1 server.cc:237] yolov3_onnx v1: UNLOADING
I0224 07:15:47.023370 1 dynamic_batch_scheduler.cc:465] Stopping dynamic-batch scheduler thread 0...
I0224 07:15:47.023416 1 onnxruntime.cc:1851] TRITONBACKEND_ModelInstanceFinalize: delete instance state
I0224 07:15:47.025484 1 onnxruntime.cc:1797] TRITONBACKEND_ModelFinalize: delete model state
I0224 07:15:47.025555 1 model_repository_manager.cc:941] OnDestroy callback() 'yolov3_onnx' version 1
I0224 07:15:47.025561 1 model_repository_manager.cc:943] successfully unloaded 'yolov3_onnx' version 1
I0224 07:15:47.025567 1 model_repository_manager.cc:730] TriggerNextAction() 'yolov3_onnx' version 1: 0
W0224 07:15:47.803079 1 metrics.cc:285] failed to get power limit for GPU 0: Not Supported
I0224 07:15:48.023777 1 model_repository_manager.cc:392] LiveBackendStates()
I0224 07:15:48.023841 1 server.cc:230] Timeout 29: Found 0 live models and 0 in-flight non-inference requests
I0224 07:15:48.023879 1 triton_backend_manager.cc:214] unloading backend 'pytorch'
I0224 07:15:48.025973 1 triton_backend_manager.cc:214] unloading backend 'tensorflow'
I0224 07:15:48.026287 1 triton_backend_manager.cc:214] unloading backend 'onnxruntime'
error: creating server: Internal - failed to load all models
`

**Comment by deadeyegoodwin** [comment]
Please provide complete console log. Errors will be indicate with lines starting with 'E'.

**Comment by tianxinyu0509** [comment]
complete console log is too long to  comment.I only find the following error:


E0225 02:15:01.631635 1 model_repository_manager.cc:1682] unexpected platform type  for .git
I0225 02:15:01.631685 1 autofill.cc:138] TensorFlow SavedModel autofill: Internal: unable to autofill for 'demo' due to no version directories
I0225 02:15:01.631697 1 autofill.cc:151] TensorFlow GraphDef autofill: Internal: unable to autofill for 'demo' due to no version directories
I0225 02:15:01.631709 1 autofill.cc:164] PyTorch autofill: Internal: unable to autofill for 'demo' due to no version directories
I0225 02:15:01.631719 1 autofill.cc:196] ONNX autofill: Internal: unable to autofill for 'demo' due to no version directories
I0225 02:15:01.631730 1 autofill.cc:209] TensorRT autofill: Internal: unable to autofill for 'demo' due to no version directories
W0225 02:15:01.631737 1 autofill.cc:248] Proceeding with simple config for now
I0225 02:15:01.631742 1 model_config_utils.cc:637] autofilled config: name: "demo"

E0225 02:15:01.631752 1 model_repository_manager.cc:1682] unexpected platform type  for demo
I0225 02:15:01.631776 1 autofill.cc:138] TensorFlow SavedModel autofill: Internal: unable to autofill for 'weights' due to no version directories
I0225 02:15:01.631791 1 autofill.cc:151] TensorFlow GraphDef autofill: Internal: unable to autofill for 'weights' due to no version directories
I0225 02:15:01.631809 1 autofill.cc:164] PyTorch autofill: Internal: unable to autofill for 'weights' due to no version directories
I0225 02:15:01.631822 1 autofill.cc:196] ONNX autofill: Internal: unable to autofill for 'weights' due to no version directories
I0225 02:15:01.631833 1 autofill.cc:209] TensorRT autofill: Internal: unable to autofill for 'weights' due to no version directories
W0225 02:15:01.631839 1 autofill.cc:248] Proceeding with simple config for now
I0225 02:15:01.631844 1 model_config_utils.cc:637] autofilled config: name: "weights"

E0225 02:15:01.631854 1 model_repository_manager.cc:1682] unexpected platform type  for weights
I0225 02:15:01.631902 1 autofill.cc:138] TensorFlow SavedModel autofill: Internal: unable to autofill for 'yolov3_onnx', unable to find savedmodel directory named 'model.savedmodel'
I0225 02:15:01.631921 1 autofill.cc:151] TensorFlow GraphDef autofill: Internal: unable to autofill for 'yolov3_onnx', unable to find graphdef file named 'model.graphdef'
I0225 02:15:01.631942 1 autofill.cc:164] PyTorch autofill: Internal: unable to autofill for 'yolov3_onnx', unable to find PyTorch file named 'model.pt'
I0225 02:15:01.631961 1 autofill.cc:196] ONNX autofill: OK: 
I0225 02:15:01.631967 1 model_config_utils.cc:637] autofilled config: name: "yolov3_onnx"

**Comment by tianxinyu0509** [comment]
Could it be that the version of TensorRT converted to OnNX does not match the Triton-Infer server?

**Comment by tianxinyu0509** [comment]
Could it be that the version of onnx converted to Onnx (such as onnx==1.5)does not match the Triton-Infer server?

**Comment by deadeyegoodwin** [comment]
What does you model repository look like. It looks like you have some extra or incorrect files and directories in it.

**Comment by tianxinyu0509** [comment]
Yes, it is. Thank you for reminding me. I have solved it.
